src/social-media-icons/_base.scss

```diff
@@ -2,127 +2,203 @@
+%social-media-icons-followus-logo-properties {
+    background-position: center center;
```

Comment:
@MaximMartel this is spaces, previous line 6 is tabs as set out in the EditConfig and SCSSLint settings